lib/renderer/init.ts

```diff
@@ -198,6 +198,7 @@ for (const preloadScript of preloadScripts) {
   } catch (error) {
     console.error(`Unable to load preload script: ${preloadScript}`)
     console.error(`${error}`)
+    console.trace();
```

Comment:
hm... it might actually be better to print out the `error` directly?

i.e. `console.error(error)` instead of stringifying it. And remove the extra `console.trace` call?

Reply:
Yeah that makes a lot more sense. I had forgotten about that functionality.